app/views/users/index.html.erb

```diff
@@ -1 +1,7 @@
-<h1>Welcome to 1-in-all Blogging site.....</h1>
\ No newline at end of file
+<div class="all-users">
+<% @users.each do |user| %>
+    <%= link_to(user) do %>
+      <%= render "shared/user_dets", user: user %>
+    <%end%>
+  <%end%>
```

Comment:
- [ ] Great job on the effort this far, however I am unable to see the views because I am unable to create users. In order to see some posts or comments, you might need to add some data to your database as described [here](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project.md#use-models-to-insert-data).

Reply:
Creating the users from UI will be in the next project with the form.